### PR TITLE
docs: fix README file

### PR DESCRIPTION
**Pull request by shibumi**
This fixes a little typo in the README + adds a better explanation for the access env variables

Related Ticket: https://issues.redhat.com/browse/OSD-10445

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ state.
 
 First make sure you have the following environment variables defined:
 ```
-AWS_ACCESS_KEY_ID           # Access Key for the aws account that should be shredded
-AWS_SECRET_ACCESS_KEY       # Secret Key for the aws account that should be shredded
+AWS_ACCESS_KEY_ID           # IAM account that has permission to shred the accounts
```
Comment (fahlmant) [inline README.md:22]: This description is not quite accurate; This var holds the Access Key ID of an Access Key pair.

File: README.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ state.
...
+AWS_SECRET_ACCESS_KEY       # Secret key for the IAM account above
```
Comment (fahlmant) [inline README.md:23]:
```suggestion
AWS_SECRET_ACCESS_KEY       # Secret key for the access key pair
```

File: README.md (hunk 1 above)
Comment (fahlmant) [reply README.md:22]: For more information, here are the docs around Access Keys https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html

**Review by fahlmant — CHANGES_REQUESTED** [review]
Comment changes are not quite accurate

**Comment by dastergon** [comment]
@fahlmant would you mind pushing your recommendations and merging? Christian is no longer working at RH.

**Comment by fahlmant** [comment]
This repo is not under active development anymore, and should be archived soon. I'm going to instead close this.

**Pull request closed without merging**